index.php

```diff
@@ -16,13 +15,17 @@
     <title>Chloe's Collector App</title>
     <link rel="stylesheet" href="https://use.typekit.net/rkk2hbc.css">
     <link rel="stylesheet" type="text/css" href="styles/style.css">
+    <html lang="en"></html>
 </head>
     <header>
         <h1>ARTISTS</h1>
     </header>
 <body>
     <?php
-       echo generateArtistHtml($artists);
+    foreach($artists as $artist) {
+        $artistsOutput = generateArtistHtml($artist);
```

Comment:
I think you should catch potential Exceptions

Reply:
done!